helpers/HTMLView.js

```diff
@@ -91,6 +95,148 @@ export function getCallbackCodeString(jsFunctionName: string, commandName: strin
 `
 }
 
+
+/**
+ * Convert a note's content to HTML and include any images as base64
+ * @author @cwhittl
+ * @param {string} content
```

Comment:
Why pass content and note? Is this different content from what the note has?

Reply:
The reason is for using it with either a whole note or selected text.
I guess I could just call the editor from the function instead of passing anything?
<img width="551" alt="image" src="https://github.com/user-attachments/assets/cde6bccd-eeef-4c09-bb37-225fe7d7ead2" />